docker-compose.yml

```
@@ -0,0 +1,107 @@
+    # Port mapping
+    ports:
+      - "8080:8080"
+    
+    # Health check (override for docker-compose)
+    healthcheck:
+      test: ["CMD", "python", "/app/healthcheck.py"]
+      interval: 30s
+      timeout: 10s
+      retries: 3
+      start_period: 60s
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)